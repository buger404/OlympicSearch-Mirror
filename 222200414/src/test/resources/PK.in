PK 中国 美利坚合众国 霹雳舞