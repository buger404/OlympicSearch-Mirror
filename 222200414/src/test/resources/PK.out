中国VS美利坚合众国 霹雳舞
gold:0/0
silver:0/0
bronze:1/1
total:1/1
-----